ansible/chapter2/pam_pwquality.yml

```diff
@@ -6,9 +6,9 @@
 
 - name: Configure pam_pwquality
   lineinfile:
-    dest: "/etc/pam.d/common-password"
+    path: "/etc/pam.d/common-password"
```

Comment:
Yeah it seems Ansible 2.3 and beyond had changed that. ```Before Ansible 2.3 this option was only usable as dest, destfile and name.``` Thanks for catching this.